### PR TITLE
Release Mock 5.6 and mock-core-configs 40.4

### PR DESCRIPTION
**Pull request by FrostyX**
Proposing a new release according to the https://github.com/rpm-software-management/mock/blob/main/docs/Releasing-Mock.md

### CODE REVIEW

**Comment by FrostyX** [comment]
Should I squash all the commits together or keep them separated?

**Comment by xsuchy** [comment]
Separated.

**Review by xsuchy — APPROVED** [review]
+1

File: docs/Release-Notes-5.6.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+- Pavel Raiskup
+- Sandro Bonazzola
+- Stephen Gallagher
+- Stewart Smith
+- Takuya Wakazono
+- Tomas Kopecek
+- Vít Ondruch
+- Yu Ming Zhu
+- zengchen1024
+- zengwei2000
```
Comment (praiskup) [inline docs/Release-Notes-5.6.md:78]: Hm, this list seems quite large to be correct...
```
(cd mock && git log mock-5.5-1..HEAD . | grep ^Author: && cd ../mock-core-configs && git log mock-core-configs-40.3-1..HEAD . | grep ^Author:) | sort | uniq
Author:     Bella Zhang <bella@cclinux.org>
Author:     David Michael <fedora.dm0@gmail.com>
Author:     Jakub Kadlcik <frostyx@email.cz>
Author:     Jani Välimaa <wally@mageia.org>
Author:     Miroslav Suchý <msuchy@redhat.com>
Author:     Nikita Gerasimov <nikitych@yandex.ru>
Author:     Pavel Raiskup <praiskup@redhat.com>
Author:     Takuya Wakazono <pastalian46@gmail.com>
```

**Pull request merged**